### PR TITLE
Added step on msi-deployment.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4231

### CODE REVIEW

**Comment by dariomws** [comment]
@LanaChin I think step 1 is not required here. You just need to remove files and registry value from the user profile that uninstalled the app, and redeploy. I did a quick test.

**Comment by LolaJacobsen** [comment]
@juturu @voltarone Please review, LMK if it's OK to merge.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/msi-deployment.md [hunk 1]
```diff
@@ -73,9 +73,13 @@ For complete guidance on how to deploy the Teams desktop app on VDI, see [Teams
 
 If a user uninstalls Teams from their user profile, the MSI installer will track that the user has uninstalled the Teams app and no longer install Teams for that user profile. To redeploy Teams for this user on a particular computer where it was uninstalled, do the following:
 
-1. Uninstall the Teams app installed for every user profile.
-2. After uninstall, delete the directory recursively under `%localappdata%\Microsoft\Teams\`.
-3. Redeploy the MSI package to that particular computer.
+> [!IMPORTANT]
+> Next steps contains information about how to modify the registry. Make sure to back up the registry before you modify it. Make sure that you know how to restore the registry if a problem occurs. For more information about how to back up, restore, and modify the registry, see [Description of the Microsoft Windows registry](https://support.microsoft.com/help/256986).
```
Comment (JohanFreelancer9) [inline Teams/msi-deployment.md]:
```suggestion
> The next steps contain information about how to modify the registry. Make sure that you back up the registry before you modify it and that you know how to restore the registry if a problem occurs. For more information about how to back up, restore, and modify the registry, see [Description of the Microsoft Windows registry](https://support.microsoft.com/help/256986).
```

**Comment by LolaJacobsen** [comment]
@LanaChin is checking with PM on this one.

**Comment by LanaChin** [comment]
Confirmed with the PMs, merged changes.

**Pull request merged**